PythonAPI/carla/setup.py

```diff
@@ -34,8 +34,9 @@ def walk(folder, file_filter='*'):
     if os.name == "posix":
         import distro
 
-        linux_distro = distro.linux_distribution()[0]
-        if linux_distro.lower() in ["ubuntu", "debian", "deepin"]:
+        supported = ["ubuntu", "debian", "deepin"]
```

Comment:
```suggestion
        supported_dists = ["ubuntu", "debian", "deepin"]
```

This variable have a different name on the `NotImplementedError` Exception message.